### PR TITLE
{lang,lib,vis}[GCCcore/6.3.0] Gdk-Pixbuf v2.36.8, libjpeg-turbo v1.5.2, NASM v2.13.01

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
depends on ~~#5037~~, ~~#5041~~

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2066.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/e7f8c7533bbf65dd35153dc897181558 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2572.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/6dd069f43b719ae795473a5956958076 for a full test report.

File: easybuild/easyconfigs/g/Gdk-Pixbuf/Gdk-Pixbuf-2.36.8-intel-2017a.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+dependencies = [
+    ('GLib', '2.53.5'),
+    ('libjpeg-turbo', '1.5.2'),
+    ('libpng', '1.6.29'),
+    ('LibTIFF', '4.0.8'),
+]
+
+configopts = "--disable-maintainer-mode --enable-debug=no --enable-introspection=yes "
+configopts += "--disable-Bsymbolic --without-gdiplus --enable-shared --enable-static"
+
```
Comment (verdurin) [inline easybuild/easyconfigs/g/Gdk-Pixbuf/Gdk-Pixbuf-2.36.8-intel-2017a.eb]: @boegel sanity check?

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 7 out of 7 (3 easyconfigs in this PR)
ca003.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/6b9765803decbb7428336903cdf98c69 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2557.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/159cc438e2174a5f816c5138dd1685bb for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
ca003.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/bb07ab74faf49d8efbf46bab9de44ddb for a full test report.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**